common/net/conn/transport/ed448/ed448_client.go

```diff
@@ -0,0 +1,99 @@
+package ed448
+
+import (
+	"github.com/pastelnetwork/gonode/common/errors"
+	"net"
+)
+
+// ClientHandshake does handshake on client side
+func (trans *Ed448) ClientHandshake(conn net.Conn) (net.Conn, error) {
```

Comment:
```suggestion
func (trans *Ed448) ClientHandshake(conn net.Conn) (newConn net.Conn, err error) {
```
we have to define for `defer` func